editor/src/components/editor/store/dispatch.tsx

```diff
@@ -674,7 +676,8 @@ function editorDispatchInner(
               ...result.unpatchedEditor.canvas,
               interactionSession: {
                 ...result.unpatchedEditor.canvas.interactionSession,
-                metadata: reconstructJSXMetadata(result.unpatchedEditor),
+                latestMetadata: reconstructJSXMetadata(result.unpatchedEditor),
+                latestAllElementProps: result.unpatchedEditor.allElementProps,
```

Comment:
heads up this is a change in behavior. on master, interactionSession.allElementProps are never updated, so it's effectively a startingAllElementProps